### ISSUE TITLE
A11y_SQL Azure Data Studio_Import ExtensionModify Columns_ScreenReader: On tab key navigation to each control present in the table there is no proper cell reference announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12978]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.2.3;#Benchmark;#NVDA;#JAWS;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3, JAWS

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Launch screen reader as NVDA and tab to checkbox in column 3

**Actual:**
On navigating to column 3 checkbox there is no announcement of column header row number or column number.
This is observed for all the column where control is present and navigated using the tab key.
The key ctrl+alt+5 as well does not announce the reference 

Issue observed with JAWS

**Expected:**
On navigating to the controls using tab key all the reference of that column number the row number and header should be announced.

**User Impact:**
The screen reader users will not get to know the cell value in the table.

**Attachment:**

[#12978_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/5401661/12978_A11y_SQL.Azure.Data.Studio_Import.ExtensionModify.Columns_ScreenReader.zip)

**Comment by aasimkhan30** [comment]
This issue was already addressed in #13216. Closing this right now

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "mport wizard " been removed


GithubTags:#Regressed:02-20-23;
